go/enclave/main/enclave.json

```diff
@@ -13,5 +13,35 @@
       "type": "hostfs",
       "readOnly": false
     }
+  ],
```

Comment:
I don't quite get it. How are the other values injected?

Reply:
Are you referring to the other flags that are not restricted ? Those values are injected via the CLI flags.